### PR TITLE
Import unit picture urls

### PR DESCRIPTION
**Pull request by japauliina**
Add importing picture URLs to the unit importer.

### CODE REVIEW

File: services/management/commands/turku_service_import/units.py [hunk 1]
```diff
@@ -194,7 +194,11 @@ def _handle_extra_info(self, obj, unit_data):
         for extra_info_data in location_data.get('lisatiedot', []):
             try:
                 koodi = extra_info_data['lisatietotyyppi'].get('koodi')
-                field_mapping = EXTRA_INFO_FIELD_MAPPING[koodi]
+                if koodi == '5':
+                    obj.picture_url = extra_info_data.get('nimi')
+                    continue
```
Comment (tuomas777) [inline services/management/commands/turku_service_import/units.py]: Is there a reason why this isn't implemented by adding the field to `EXTRA_INFO_FIELD_MAPPING` ?

Comment (frwickst) [reply services/management/commands/turku_service_import/units.py]: I agree here, seems like adding
```
'6': {'nimi': 'picture_url'},
```
would do the same thing here I think :)

Comment (tuomas777) [reply services/management/commands/turku_service_import/units.py]: `'5': {'nimi': 'picture_url'},` to be exact 😉

Comment (japauliina) [reply services/management/commands/turku_service_import/units.py]: Yes makes much more sense, I don't know how I missed that - now fixed!

Comment (tuomas777) [reply services/management/commands/turku_service_import/units.py]: well, can't blame you, the thing is pretty cryptic and of course not commented. This was a bit easier for me as I'm the original author 😄

**Review by tuomas777 — APPROVED** [review]
👍

**Pull request merged**